permissions/plugin-tricentis-ci.yml

```diff
@@ -0,0 +1,8 @@
+---
+name: "tricentis-ci"
+github: "jenkinsci/tricentis-ci-plugin"
+paths:
+- "org/jenkins-ci/plugins/tricentis-ci"
```

Comment:
Conflicts with https://github.com/jenkinsci/tricentis-ci-plugin/blob/48fb761949ff1edf6d706ae4d829118e65b02701/pom.xml#L11

Reply:
Thanks for the review @daniel-beck and sorry for this stupid typo, I have fixed the path according to the groupId in the pom.xml.